frontend/src/components/v1/DataTable.css

```
@@ -103,3 +103,49 @@
+.data-table--loading {
+  overflow: hidden;
+}
+
+.data-table-loading__status {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)